client/me/purchases/credit-cards/credit-card-delete.jsx

```diff
@@ -17,15 +18,17 @@ const CreditCardDelete = React.createClass( {
 		this.props
 			.deleteStoredCard( this.props.card )
 			.then( () => {
-				this.props.successNotice( this.translate( 'Card deleted successfully' ) );
+				this.props.successNotice( this.props.translate( 'Card deleted successfully' ) );
 			} )
 			.catch( error => {
 				this.props.errorNotice( error.message );
 			} );
 	},
 
 	renderDeleteButton: function() {
-		const text = this.props.isDeleting ? this.translate( 'Deleting ' ) : this.translate( 'Delete' );
+		const text = this.props.isDeleting
+			? this.props.translate( 'Deleting ' )
```

Comment:
I think that would be a good idea. Note there are [translations for `Deleting...`](https://translate.wordpress.com/projects/wpcom/fr/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=129436&filters%5Btranslation_id%5D=6104081) in GlotPress so using the later will make sure users are served with valid translations.